templates/web/base/admin/category_translation_edit.html

```diff
@@ -0,0 +1,31 @@
+[% INCLUDE 'admin/header.html' title=tprintf(loc('Edit %s Translation for %s'), language_name, contact.category) -%]
+
+<form method="post" action="[% c.uri_for('body', body_id, contact.category ) %]" enctype="application/x-www-form-urlencoded" accept-charset="utf-8" id="category_edit">
+    <input type="hidden" name="translation_id" value="[% translation.id| html %]" >
+    <input type="hidden" name="token" value="[% csrf_token %]" >
+    <input type="hidden" name="posted" value="edit_translation" >
+    <input type="hidden" name="language" value="[% translation.lang| html %]" >
+    <p>
+    <strong>Original</strong>: [% contact.category | html %]
```

Comment:
Same comments for here (is it worth sharing these two templates, they seem very similar).